src/univariate/continuous/normal.jl

```diff
@@ -27,7 +27,10 @@ struct Normal{T<:Real} <: ContinuousUnivariateDistribution
     μ::T
     σ::T
 
-    Normal{T}(μ, σ) where {T} = (@check_args(Normal, σ > zero(σ)); new{T}(μ, σ))
+    function Normal{T}(μ, σ) where {T}
```

Comment:
Mention the behavior in the documentation of `Normal`

Reply:
also, some tests for the std=0 case should be added

Reply:
Fixed!